shared_kv_storage/kv_storage/interface/shared_kv_interface.py

```diff
@@ -86,6 +86,61 @@ def lookup(self, table):
            - table is the name of lmdb table.
         """
         pass
+
+# ---------------------------------------------------------------------------------------------------
+    @abstractmethod
+    def csv_append(self, table, key, value):
+        """
+        Function to update a key-value pair in a lmdb table that holds
+        comma-separated strings as value. This function adds a string
+        to the end of the comma-separated value.
+
+        Parameters:
+           @param table - Name of the lmdb table in which key-value pair
+                          needs to be updated.
+           @param key - The primary key of the table.
+           @param value - The value that needs to be appended to the existing
+                          value(comma-separated) corresponding to the key.
+        """
+        pass
+
+# ---------------------------------------------------------------------------------------------------
+    @abstractmethod
+    def csv_prepend(self, table, key, value):
```

Comment:
Do we need to add these csv functions to KV store interface? Better add them lmdb implementation class itself.

Reply:
Agree @Ram-srini . csv is a file format. shared_kv_interface.py is more generic set/get/lookup database functions. CSV specific functions should be in a different interface.